tonatona-sample/src/Tonatona/Sample.hs

```diff
@@ -133,22 +134,56 @@ instance TonaServant.HasConfig Config where
 -- Shared
 
 data Shared = Shared
-  { tonaDb :: TonaDb.Shared
+  { tonaDb :: SharedDb
   , tonaLogger :: TonaLogger.Shared
   }
 
+
+data SharedDb
+  = SharedSqlite TonaDbSqlite.Shared
+  | SharedPostgres TonaDbPostgres.Shared
```

Comment:
Because we choose at runtime to use either Sqlite or Postgresql, we need to have this data type that can have either an sqlite connection or a postgresql connection.

This feature would be nice to have in goat guardian.  It would be nice if the operator of goat guardian could pass a flag on the command line to decide to use either sqlite or postgresql.